test/unit/mocks.js

```diff
@@ -29,6 +29,38 @@ const contentTypeMock = {
       type: 'Text',
       localized: true,
       required: false
+    },
```

Comment:
would probably be valid to have the `type: 'ResourceLink` here as well

Reply:
That one is a `Text` field so it's not a `ResourceLink`. Are we talking about the same thing?

```
    {
      id: 'fieldid',
      name: 'fieldname',
      type: 'Text',
      localized: true,
      required: false
    }
```

Reply:
hm, I was just mentioning that this mock doesn't have a field `type: 'ResourceLink'`, which might be useful to have in it for testing:
(my comment was just on an random line)

```
 {
      id: 'fieldid',
      name: 'fieldname',
      type: 'ResourceLink',
      localized: true,
      required: false
    }

```

Reply:
ok I understand now. I'll do it 👍